orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -92,6 +116,30 @@ class ORTTrainerOptions(object):
                         'enable_adasum' : {
                             'type' : 'boolean',
                             'default' : False
+                        },
+                        'sliced_schema' : {
+                            'type': 'dict',
+                            'default' : {},
+                            'required': False,
+                            # the keys in `sliced_schema` are strings.
+                            'keysrules': {'type': 'string'},
+                            # the values in `sliced_schema` are lists of integers.
+                            'valuesrules': {
+                                'type': 'list',
+                                'schema': {'type': 'integer'}
+                            }
+                        },
+                        'sliced_axes' : {
+                            'type': 'dict',
```

Comment:
Similar to my previous comment. Define the dict schema inside `schema`

Reply:
Sure.

---
In reply to: [542786782](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r542786782) [](ancestors = 542786782)